src/main/java/com/internship/training/StreamsChallenge.java

```diff
@@ -2,14 +2,20 @@
 
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.xml.internal.ws.policy.privateutil.PolicyUtils;
 
 import java.io.File;
 import java.io.IOException;
-import java.util.List;
+import java.util.*;
```

Comment:
There are some issues with this kind of import. As a new task do a quick research on why using * on the imports can be bad in some cases an answer in this comment.

Reply:
1) If you import two library, which have classes with the same name, it will generate namespace collision.
2)Wildcard imports are a little bit slower

Specific imports improve the readability of code because they make it easier to see where a class is coming from.